recipes/coverage/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "4.2" %}
```

Comment:
Please see this [discussion]( https://github.com/conda-forge/numpy-feedstock/issues/10 ). The problem with not doing it this way is someone may be tempted to clean up the feedstock to match version order after the fact. That seems far more disastrous to me then asking for a simple downgrade and addressing upgrades in the feedstock.